package.json

```diff
@@ -21,21 +21,23 @@
   },
   "dependencies": {
     "@kth/reqvars": "^2.0.1",
-    "bunyan": "^1.8.12",
+    "bluebird": "^3.7.2",
+    "bunyan": "^1.8.13",
     "dotenv": "^5.0.1",
     "express": "^4.17.0",
     "got": "^9.6.0",
     "kth-canvas-api": "^0.15.7",
```

Comment:
Isn't this dependency now deprecated and we should switch to @kth/canvas-api instead? Since this PR is about updating deps, isn't now a good idea to do that?